scripts/nolints.json

```diff
@@ -49,4 +49,5 @@
  ["docBlame", "String.foldrAuxWF.loop"],
  ["docBlame", "String.replaceWF.loop"],
  ["docBlame", "String.substrEqWF.loop"],
- ["docBlame", "Substring.splitOnWF.loop"]]
\ No newline at end of file
+ ["docBlame", "Substring.splitOnWF.loop"],
+ ["simpNF", "List.get_cons_zero"]]
```

Comment:
this should be a `nolint` on the relevant declaration

Reply:
Oops. `nolint` is not available in that file. However this is a leftover from an earlier draft in which `List.get_cons_zero` said something different. Now the linter doesn't complain anyway.

The remaining problem stemming from this lemma is `Fin.zero_eta` which I've currently left stranded at the top of `Std.Data.List.Init.Lemmas`.